Dockerfile

```diff
@@ -26,6 +26,11 @@ FROM python:2.7
 #
 # set up environment
 #
+ENV TINI_VERSION v0.10.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+ENTRYPOINT ["/tini", "--"]
```

Comment:
I didn't dig much into it, but this is the best explanation I found https://github.com/krallin/tini/issues/8 .